README.md

```diff
@@ -7,33 +7,30 @@ compiled for the wasm32-wasi target.
 
 ## Getting started
 
-To run this builds for WASM/WASI you will need to install [WebAssembly/wasi-sdk](https://github.com/WebAssembly/wasi-sdk) in advance. Then export the location as `WASI_SDK_ROOT`.
+All you need in order to run these builds is to have `docker` or
+`podman` available in your system. You can execute the following
+`Makefile` targets:
 
-First ensure you have the build tools required to build the respective language runtime or standalone library. These could be `autoconf`, `libtool`, `make`, etc.
+- `php/php-7.3.33`
+    - Resulting binaries are placed in `php/build-output/php/php-7.3.33/bin`.
 
-For the current version of our build orchestration scripts we use `bash`
+- `php/php-7.4.32`
+    - Resulting binaries are placed in `php/build-output/php/php-7.4.32/bin`.
```

Comment:
@ereslibre I would avoid auto-splitting markdown lines. I know this is done automatically by some IDEs but it's not a common action. Most of the time, a README file will be read in the GitHub repo, so some styles will be applied. 

However, when you're editing a markdown file and your editor is not configured in this way, calculating it manually is difficult and you end up with random changes in these files in the future. My 2 cents, but not a blocker here.

Reply:
Ugh yeah, it's my default editor settings. Let me fix that.